examples/js/src/main/kotlin/com/js/sample/Handler.kt

```diff
@@ -0,0 +1,20 @@
+package com.js.sample
+
+import com.kotlin.aws.js.runtime.client.LambdaHTTPClient
+import com.kotlin.aws.js.runtime.objects.LambdaContext
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+
+class Handler {
+    suspend fun handler(context: LambdaContext, request: String) {
```

Comment:
probably it should be a function with the same params as in default nodejs runtime. [Example](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html)